lib/env.dart

```diff
@@ -4,4 +4,12 @@ class Env {
   static String get restApiEndpoint {
     return FlutterConfig.get('REST_API_ENDPOINT');
   }
+
+  static String get key {
+    return FlutterConfig.get('KEY');
```

Comment:
what do you think about the remaining `key` to `authClientID`? The current one is kindly ambiguous 🤔